pymess/backend/dialer/daktela.py

```diff
@@ -53,6 +55,9 @@ def _update_dialer_states(self, messages):
                 timeout=settings.DIALER_DAKTELA.TIMEOUT
             ).get(client_url)
             resp_json = response.json()
+            if response.status_code != 200 and resp_json.get('error'):
+                self._update_message_with_error(message, error_message=resp_json.get('error'), is_final_state=False)
```

Comment:
do not halt status checking for the following messages. is_final_state=False to check it again when client had temporary issue.

Reply:
když dialer zprávu nebude znát tak to bude navolávat donekoečna?

Reply:
this method is to check status of already sent messages. If we send a record which does not have a final state it will check the status again and again. I think it's a correct way to do. Either it's temporary downtime of Daktela or there is some error to fix on our side.

Reply:
The status of sent records which are not confirmed by Daktela will be excluded and won't be checked.

Reply:
V překladu, bojím se toho, aby se to nevolalo u nějakých zpráv donekonečna. Mělo by to mít nějaké počítadlo a zkusit to třeba jen 5x. Přidej ho tam prosím, když to bude popáté tak se to nastaví jako final. Koukni jak je to u e-mailů. Tam je number_of_send_attempts

Reply:
ok, added